### PR TITLE
feat: unify description to one standard

### LINKED ISSUE #134: Unifying the description to one standard
All of CLI describe a arguments with comma and of line or text in uppercase

I suggest to unifying the description, to this rules: 
- First litter is capital letter
- Text in lowercase
- Without comma and of line

### PR DESCRIPTION
**Pull request by ifedchankau**
Unify description to one standard

Closes #134

### CODE REVIEW

File: src/arguments.js [hunk 1]
```diff
@@ -15,14 +15,28 @@ const getArgumentObject = (object, context) => {
...
+ * @param {string} description - argument description
+ * @return {string} description - unified description
+ */
+const unifyDescription = (description) => {
+    description = description.trim();
+    description = description.charAt(0).toUpperCase() + description.slice(1);
+    if (description.charAt(description.length - 1) === '.') {
+        description = description.slice(0, description.length - 1);
+    }
+    return description;
```
Comment (romanrostislavovich) [inline src/arguments.js]: Please use pure function.

File: src/arguments.js [hunk 2]
```diff
@@ -15,14 +15,28 @@ const getArgumentObject = (object, context) => {
...
     return argument;
 };
 
+/**
+ * Unify description to one standard
+ * @param {string} description - argument description
+ * @return {string} description - unified description
+ */
+const unifyDescription = (description) => {
+    description = description.trim();
```
Comment (romanrostislavovich) [inline src/arguments.js]: I think we need to validate input description by `string` and `length` (if description is empty)

File: src/arguments.js [hunk 3]
```diff
@@ -15,14 +15,28 @@ const getArgumentObject = (object, context) => {
...
+    description = description.charAt(0).toUpperCase() + description.slice(1);
+    if (description.charAt(description.length - 1) === '.') {
```
Comment (romanrostislavovich) [inline src/arguments.js]: I'm not sure but we can use ternary operator for this

File: src/arguments.js (hunk 2 above)
Comment (romanrostislavovich) [reply src/arguments.js]: Also we can to combine `trim` with another function for `description`

File: src/arguments.js (hunk 1 above)
Comment (ifedchankau) [reply src/arguments.js]: @svoboda-rabstvo done.

File: src/arguments.js (hunk 2 above)
Comment (ifedchankau) [reply src/arguments.js]: @svoboda-rabstvo i've add validate. Trim is better to leave separate.

File: src/arguments.js (hunk 3 above)
Comment (ifedchankau) [reply src/arguments.js]: @svoboda-rabstvo done.

**Review by romanrostislavovich — DISMISSED** [review]
LGTM

**Review by romanrostislavovich — APPROVED** [review]
+1

**Pull request merged**